src/main/java/org/sagebionetworks/bridge/hibernate/HibernateAdherenceReportDao.java

```diff
@@ -53,23 +59,39 @@ public void saveWeeklyAdherenceReport(WeeklyAdherenceReport report) {
         WhereClauseBuilder where = builder.startWhere(AND);
         where.append("h.appId = :appId", APP_ID_FIELD, appId);
         where.append("h.studyId = :studyId", STUDY_ID_FIELD, studyId);
-        if (complianceUnder != null) {
-            where.append("weeklyAdherencePercent < :complianceUnder", COMPLIANCE_UNDER_FIELD, complianceUnder);
+        if (search.getAdherenceMin() > 0) {
+            where.append("weeklyAdherencePercent >= :adherenceMin", "adherenceMin", search.getAdherenceMin());    
+        }
+        if (search.getAdherenceMax() < 100) {
+            where.append("weeklyAdherencePercent <= :adherenceMax", "adherenceMax", search.getAdherenceMax());    
+        }
```

Comment:
This is minor, but the adherence min/max key parameters could be replaced with their constants to be consistent with the other append() calls.